phing/tasks/setup.xml

```diff
@@ -245,21 +246,50 @@
       <option name="entity-updates"></option>
     </drush>
 
-    <!-- Import configuration. -->
+    <!-- This will call one of the following:
+      setup:config-import:features
+      setup:config-import:core-only
+      setup:config-import:config-split
+    -->
+    <phingcall target="setup:config-import:${cm.strategy}"/>
+
+    <!-- Rebuild caches. -->
+    <drush command="cr" alias="${drush.alias}" passthru="false"/>
+  </target>
+
+  <target name="setup:config-import:core-only">
+    <drush command="config-import" assume="yes" alias="${drush.alias}" passthru="false">
+      <param>${cm.core.key}</param>
+    </drush>
+  </target>
+
+  <target name="setup:config-import:config-split">
+    <drush command="en" assume="yes" alias="${drush.alias}" passthru="false">
+      <param>config_split</param>
+    </drush>
     <if>
-      <equals arg1="${cm.core.partial}" arg2="true"/>
+      <isset property="env.AH_SITE_ENVIRONMENT"/>
       <then>
-        <drush command="config-import" assume="yes" alias="${drush.alias}" passthru="false">
-          <option name="partial"></option>
-          <param>${cm.core.config-dir.key}</param>
+        <mkdir dir="${cm.core.path}/${env.AH_SITE_ENVIRONMENT}"/>
+        <drush command="config-split-import" assume="yes" alias="${drush.alias}">
+          <!-- Cloud environments require vcs parameter. -->
+          <param>vcs</param>
+          <option name="split">${environment}</option>
```

Comment:
@arknoll Does it have to be done that way? I'd much rather use a param than write to settings.php.